components/inputs/input-checkbox.js

```diff
@@ -85,9 +130,16 @@ class InputCheckbox extends RtlMixin(LitElement) {
 					margin-left: 0;
 					margin-right: 0;
 				}
+				:host([skeleton]) .d2l-input-checkbox-text.d2l-skeletize::before {
+					bottom: 0.3rem;
+					top: 0.3rem;
```

Comment:
Since checkboxes don't use one of our standard typography styles, I needed to explicitly adjust the top/bottom to get the right cap height on the skeleton.